brands/amenity/cinema.json

```
@@ -295,8 +339,34 @@
       "name": "Haut et Court"
     }
   },
+  "amenity/cinema|Les Écrans de Paris": {
+    "locationSet": {
+      "include": ["fr"]
+    },
+    "tags": {
+      "amenity": "cinema",
+      "brand": "Les Écrans de Paris",
+      "brand:wikidata": "Q3236510",
+      "brand:wikipedia": "fr:Les Écrans de Paris",
+      "name": "Les Écrans de Paris"
+    }
+  },
+  "amenity/cinema|Groupe Majestic Cinemas": {
+    "locationSet": {
+      "include": ["fr"]
+    },
+    "tags": {
+      "amenity": "cinema",
+      "brand": "Les Écrans de Paris",
```

Looks like this still has the brand from the other cinema?  It should probably just say "Majestic"
```suggestion
      "brand": "Majestic",
```